CHANGES.md

```diff
@@ -15,12 +15,19 @@ ones in. -->
 
 ### Enhancements
 
+[#5475](https://github.com/cylc/cylc-flow/pull/5475) - much faster computation
+of the visualization window around active tasks (at the cost, for now, of not
+showing non-active "cousin" nodes).
```

Comment:
??? What the hell. I don't know why this is appearing here. (No time to investigate now).

Reply:
Added in c17529a282bae4e88c4373c35516c09f826c95a8. IMHO it's a good addition as #5475 was missing a changelog entry

Reply:
Hmm, I thought I had pushed that as a hot fix to the 8.1.x branch. Must have got my wires crossed somewhere.

Reply:
A-ha, I had added the commit to my local 8.1.x branch but forgot to push it to GitHub. Then I rebased this branch on to 8.1.x locally and pushed this branch to GitHub.  That explains it 😌